code-experiments/build/matlab/my_optimizer.m

```diff
@@ -1,9 +1,13 @@
-function my_optimizer (problem, lower_bounds, upper_bounds, budget)
+function my_optimizer (problem, lower_bounds, upper_bounds, num_integer_vars, num_constraints, budget)
 n = length(lower_bounds);
 delta = upper_bounds - lower_bounds;
 for i= 1:budget
     x = lower_bounds + rand(1,n) .* delta;
-    if cocoProblemGetNumberOfConstraints(problem) > 0
+    % Round the variable values that need to be integer
```

Comment:
Is there a better way to explicitly show that problems can also be mixed-integer (this code makes little sense)?

Reply:
> @kostasvar Can you produce the new data for the regression tests?

They are uploaded now.